packages/esm-patient-registration-app/src/patient-registration/field/address/address-hierarchy.component.tsx

```diff
@@ -1,45 +1,42 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState, useContext } from 'react';
+import styles from '../field.scss';
 import { useTranslation } from 'react-i18next';
-import { Input } from '../../input/basic-input/input/input.component';
 import { ResourcesContext } from '../../../offline.resources';
-import styles from '../field.scss';
+import { ComboInput } from '../../input/comboinput/comboinput.component';
 
+export function getFieldValue(field: string, doc: XMLDocument) {
+  const fieldElement = doc.getElementsByName(field)[0];
+  return fieldElement ? fieldElement.getAttribute('value') : null;
+}
 function parseString(xmlDockAsString: string) {
   const parser = new DOMParser();
   return parser.parseFromString(xmlDockAsString, 'text/xml');
 }
-
 function getTagAsDocument(tagName: string, template: XMLDocument) {
   const tmp = template.getElementsByTagName(tagName)[0];
   return tmp ? parseString(tmp.outerHTML) : parseString('');
 }
 
-function getFieldValue(field: string, doc: XMLDocument) {
-  const fieldElement = doc.getElementsByTagName(field)[0];
-  return fieldElement ? fieldElement.getAttribute('value') : null;
-}
-
-export const AddressField: React.FC = () => {
-  const { addressTemplate } = useContext(ResourcesContext);
-  const [addressFields, setAddressFields] = useState([]);
+export const AddressHierarchy: React.FC = () => {
+  const [comboboxlist] = useState([]);
```

Comment:
I feel like you don't need this, but if your not happy with throwing away the variable then:

`comboboxlist` --> `comboboxList`